### ISSUE TITLE
Smoketest: Bring back data migration tests again

### ISSUE THREAD

**Issue by joaomoreno** [Microsoft/vscode#48107]

**Comment by bpasero** [comment]
@joaomoreno isn't the migration test starting stable first and then insiders to see if the UI state is the same? Given this is about starting 2 different milestones of the product, I do not see how we could test this with unit tests.

**Comment by joaomoreno** [comment]
I don't know where that notion of stable+insiders came from, but I think it is wrong. It should be two consecutive stable versions, right? The flavour should remain the same.

I guess we can have the smoketest download a version -1 of the current version off the internet and use it for the tests.

**Comment by bpasero** [comment]
@joaomoreno yeah it is stable vs stable-1, but typically when we run this test we do not have a stable release yet, so we use the last insiders before declaring that one stable.
